<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWaveSpectrumDialog</class>
 <widget class="QWidget" name="ConfigWaveSpectrumDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>642</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wave Spectrum</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,8">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="10,1,1,1">
      <item>
       <widget class="QTableWidget" name="table_SpectralAnalysisPara">
        <property name="font">
         <font>
          <family>宋体</family>
         </font>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="columnCount">
         <number>10</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>260</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>WaveSpectrumType</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>SignificantWaveHeight(m)</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>WaveDirectionaAngle(°)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>LowCutoffFreq(rad/s)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>HighCutoffFreq(rad/s)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>WavesDiscreteRegularWaveNum</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>WavesDiscreteNum</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>RandomSeedNum</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>PeakSpectralPeriod(s)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>SpectralWindFactor(γ)</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>ITTCSingleParaSpectrum</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>6.9</string>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>180</string>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string>0.01</string>
         </property>
        </item>
        <item row="0" column="4">
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item row="0" column="5">
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item row="0" column="6">
         <property name="text">
          <string>80</string>
         </property>
        </item>
        <item row="0" column="7">
         <property name="text">
          <string>200</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="6" column="1">
         <widget class="SelfDefLineEdit" name="Edit_SpectrumHeartCycle"/>
        </item>
        <item row="1" column="1">
         <widget class="SelfDefLineEdit" name="Edit_ShowAngleCbo"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_WaveSpectrumType">
          <property name="text">
           <string>WaveSpectrumType</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="WaveSpectrumTypeCbo">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <item>
           <property name="text">
            <string>ITTCSingleParaSpectrum</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ITTCDoubleParaSpectrum</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>JONSWAPWaveSpectrum</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_SpectralWindFactor">
          <property name="text">
           <string>SpectralWindFactor(γ)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="SelfDefLineEdit" name="Edit_SignificantWaveHeight"/>
        </item>
        <item row="7" column="1">
         <widget class="SelfDefLineEdit" name="Edit_SpectralWindFactor"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_HighCutoffFreq">
          <property name="text">
           <string>HighCutoffFreq(rad/s)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="SelfDefLineEdit" name="lineEdit_HighCutoffFreq"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_LowCutoffFreq">
          <property name="text">
           <string>LowCutoffFreq(rad/s)</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_SpectrumHeartCycle">
          <property name="text">
           <string>SpectrumHeartCycle(T1/s)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_WaveDirectionAngle">
          <property name="text">
           <string>WaveDirectionAngle(°)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_SignificantWaveHeight">
          <property name="text">
           <string>SignificantWaveHeight(m)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="SelfDefLineEdit" name="lineEdit_LowCutoffFreq"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>GeneralRandomWaveNumSetting</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="randomFirstLabel">
             <property name="text">
              <string>WavesDiscreteRegularWaveNum</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="randomsecondLabel">
             <property name="text">
              <string>WavesDiscreteNum</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="randomthreeLabel">
             <property name="text">
              <string>RandomSeedNum</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="SelfDefLineEdit" name="randomFirstLineEdit"/>
           </item>
           <item>
            <widget class="SelfDefLineEdit" name="randomSecondLineEdit"/>
           </item>
           <item>
            <widget class="SelfDefLineEdit" name="randomThreeLineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_29">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="AddBtn_Spectral">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ModifyBtn_Spectral">
          <property name="text">
           <string>Modify</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DeleteBtn_Spectral">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="curveWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutCurve"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelfDefLineEdit</class>
   <extends>QWidget</extends>
   <header>selfdeflineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
